ci/L0_backend_vllm/test.sh

```diff
@@ -30,6 +30,8 @@ SUBTESTS="accuracy_test request_cancellation enabled_stream vllm_backend"
 
 python3 -m pip install --upgrade pip && pip3 install tritonclient[grpc]
 
+cp ../../src/model.py /opt/tritonserver/backends/vllm/
```

Comment:
Please, remove it. In our ci, we build container with `model.py` from a specified branch already present in `/opt/tritonserver/backends/vllm/`